### PR TITLE
Link a user to a constituency.

### PR DESCRIPTION
**Pull request by bmarika**

### CODE REVIEW

File: common/serializers/serializer_declarations.py [hunk 1]
```diff
@@ -159,3 +160,15 @@ class FilteringSummariesSerializer(serializers.Serializer):
     facility_type = FilteringOptionsSerializer(many=True)
     service_category = FilteringOptionsSerializer(many=True)
     operation_status = FilteringOptionsSerializer(many=True)
+
+
+class UserConstituencySerializer(
+        AbstractFieldsMixin, serializers.ModelSerializer):
+    user_email = serializers.ReadOnlyField(source='user.email')
+    user_name = serializers.ReadOnlyField(source='user.get_full_name')
+    constituenncy_name = serializers.ReadOnlyField(source='constituenncy.name')
```
Comment (ngash) [inline common/serializers/serializer_declarations.py]: Typo in `constituency_name` (and the next two lines)

File: facilities/migrations/0001_initial.py [hunk 2]
```diff
@@ -199,12 +226,12 @@ class Migration(migrations.Migration):
                 ('sub_division', models.CharField(help_text=b'This is a further division of the facility type e.g Hospitals can be further divided into District hispitals and Provincial Hospitals.', max_length=100, null=True, blank=True)),
                 ('created_by', models.ForeignKey(related_name='+', on_delete=django.db.models.deletion.PROTECT, default=common.models.base.get_default_system_user_id, to=settings.AUTH_USER_MODEL)),
                 ('preceding', models.ForeignKey(related_name='preceding_type', blank=True, to='facilities.FacilityType', help_text=b'The facility type that comes before this type', null=True)),
-                ('suceedding', models.ForeignKey(related_name='suceedding_type', blank=True, to='facilities.FacilityType', help_text=b'The facility type that comes afetr this type', null=True)),
```
Comment (ngash) [inline facilities/migrations/0001_initial.py:202]: why was this removed?

**Comment by bmarika** [comment]
@ngash merge hii

**Comment by ngash** [comment]
the `static/.gitikeep` file was required to retain the `static` folder in version control

File: facilities/migrations/0001_initial.py (hunk 2 above)
Comment (bmarika) [inline facilities/migrations/0001_initial.py:202]: This was removed a long time ago. I encountered a cyclic dependency while creating data for the model.
